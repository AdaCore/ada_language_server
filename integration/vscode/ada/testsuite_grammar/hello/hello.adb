-- SYNTAX TEST "source.ada" "simple test"
-- simple test

procedure Hello is
begin
   null;
end Hello;
