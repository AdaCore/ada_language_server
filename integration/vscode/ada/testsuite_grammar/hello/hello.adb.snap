>-- SYNTAX TEST "source.ada" "simple test"
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ada comment.line.double-dash.ada
>-- simple test
#^^^^^^^^^^^^^^ source.ada comment.line.double-dash.ada
>
>procedure Hello is
#^^^^^^^^^ source.ada keyword.ada
#         ^ source.ada
#          ^^^^^ source.ada entity.name.function.ada
#               ^ source.ada
#                ^^ source.ada keyword.ada
>begin
#^^^^^ source.ada keyword.ada
>   null;
#^^^ source.ada
#   ^^^^ source.ada meta.statement.null.ada keyword.ada
#       ^ source.ada meta.statement.null.ada punctuation.ada
>end Hello;
#^^^ source.ada keyword.ada
#   ^ source.ada
#    ^^^^^ source.ada entity.name.function.ada
#         ^ source.ada punctuation.ada
>